mjx/mujoco/mjx/_src/types.py

```diff
@@ -1467,3 +1477,4 @@ class Data(PyTreeNode):
   _qM_sparse: jax.Array = _restricted_to('mjx')  # pylint:disable=invalid-name
   _qLD_sparse: jax.Array = _restricted_to('mjx')  # pylint:disable=invalid-name
   _qLDiagInv_sparse: jax.Array = _restricted_to('mjx')  # pylint:disable=invalid-name
+  energy: jax.Array
```

Comment:
We try to match the field order in mjdata.h - can you move this below time?  And move in the docstring too.